templates/taxi/car_detail.html

```diff
@@ -10,6 +10,14 @@ <h1>
     <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
       Update
     </a>
+    <form action="" method="post">
+      {% csrf_token %}
+      {% if user in car.drivers.all %}
+        <input type="submit" class="btn btn-danger link-to-page" value="Delete me from this car">
+      {% else %}
+        <input type="submit" class="btn btn-success link-to-page" value="Assign me to this car">
+      {% endif %}
+    </form>
   </h1>
```

Comment:
It is a code style issue. The form and the buttons should not be inside the h1 tag. It is better to move them outside of the h1 tag and use CSS for styling.